dist/entrypoint.sh

```diff
@@ -98,21 +95,13 @@ fi
 #
 if [ $UNITY_EXIT_CODE -eq 0 ]; then
   # Activation was a success
-  echo ""
-  echo "###########################"
-  echo "#   Activation complete   #"
-  echo "###########################"
-  echo ""
+  echo "Activation complete."
 else
-  echo ""
-  echo "###########################"
-  echo "#         Failure         #"
-  echo "###########################"
-  echo ""
-  echo "Please note that the exit code is not very descriptive."
-  echo "Most likely it will not help you solve the issue."
-  echo ""
-  echo "To find the reason for failure: please search for errors in the log above."
```

Comment:
Prefer to keep this "To find the reason"  part, as it reduced the amount of questions on discord by quite a bit.

Reply:
Added back